scala/private/phases/phase_collect_srcjars.bzl

```diff
@@ -3,12 +3,15 @@
 #
 # DOCUMENT THIS
 #
-load(
-    "@io_bazel_rules_scala//scala/private:common.bzl",
-    "collect_srcjars",
-)
 
 def phase_collect_srcjars(ctx, p):
     # This will be used to pick up srcjars from non-scala library
     # targets (like thrift code generation)
-    return collect_srcjars(ctx.attr.deps)
+    return _collect_srcjars(ctx.attr.deps)
```

Comment:
Same here, why not
```
def phase_collect_srcjars(ctx, p):
    # This will be used to pick up srcjars from non-scala library
    # targets (like thrift code generation)
    srcjars = []
    for target in ctx.attr.deps:
        if hasattr(target, "srcjars"):
            srcjars.append(target.srcjars.srcjar)
    return depset(srcjars)
```

Reply:
fixed